<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyForm</class>
 <widget class="QWidget" name="MyForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>716</width>
    <height>420</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QLabel" name="namelabel">
     <property name="text">
      <string>name</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="nameEdit"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>age</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="ageEdit"/>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QPushButton" name="okBtn">
     <property name="text">
      <string>ok</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>js alert</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <layout class="QGridLayout" name="gridLayout"/>
   </item>
   <item row="3" column="2">
    <widget class="QWidget" name="widget" native="true">
     <widget class="QPlainTextEdit" name="jsEdit">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>20</y>
        <width>201</width>
        <height>111</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="callJsBtn">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>140</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>callJsBtn</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
